<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QT_ASM_ConverterClass</class>
 <widget class="QMainWindow" name="QT_ASM_ConverterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT_ASM_Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="X86_radioButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X86</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="X64_radioButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X64</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ASM_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ASM_TO_HEX_pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ASM TO HEX</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Hex_textEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="Point_checkBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>使用地址</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QT_ASM_Converter.qrc"/>
 </resources>
 <connections/>
</ui>
